lib/tasks/test.rake

```diff
@@ -0,0 +1,4 @@
+desc "Run all tests (rspec, rubocop, sass-lint)"
+task :sass_lint do
+  system("yarn && node_modules/.bin/sass-lint -vq")
```

Comment:
Why do we need to yarn first?

Reply:
because ./node_modules is not tracked in git, and so every environment must run `yarn` to get it.  Unless I'm wrong!